src/collections/_documentation/workflow/releases/health.md

```diff
@@ -0,0 +1,137 @@
+
+## Releases Index
+
+The releases index page provides a high-level view of:
+
+- Each release version
+- The associated project
+- The authors of each commit
+- The percentage of crash-free users
+- The percentage of crash-free sessions
+
+Each release links to one project. If a release has multiple projects, Sentry will duplicate the release data in relation to each project. The data will present adoption of the release from the most recent 24 hours.
+
+[{% asset releases/release_index.png alt="View of the release index page showing each version of projects related to the release and the project details." %}]({% asset releases/release_index.png @path %})
+
+## Time Range Change
+
+When you change the date range, Sentry recalculates the values. Sentry doesn't provide the average value of the daily crash-free users or daily crash-free sessions, but the actual value of those metrics calculated for the selected time period. The number can significantly change based on the selected time range. For example:
+
+Let's say you selected the 24-hour time range, have seven users that are using your application for seven days, and each day one of them experiences a crash. The Release Index page will show you that every day there were 85.7% crash-free users on the release.
+
+In contrast, let's say you selected the seven-day time range, have seven users, and each day a different user experiences a crash. By the end of the week, the Release Index page will show you the project had 0% crash-free users.
+
+## Release Details
+
+The release details page focuses on an individual release. Elements of the release are presented in visual trends for crashes and sessions, specifics regarding each issue, and commit author breakdowns.
+
+[{% asset releases/release_details.png alt="View of the release details page showing an overview graph, files changed, and other details." %}]({% asset releases/release_details.png @path %})
+
+For more information about Sentry's Mobile features, see the full documentation for [Android SDK]({%- link _documentation/platforms/android/index.md -%}) and [iOS SDK](docs.sentry.io/platforms/cocoa/).
+
+## Getting Started
+
+Currently, we only support the health functionality for Android and iOS applications.
+
+### Android
+
+To benefit from the health data provided in Releases v2, you must use at least 2.1.0 of the Android SDK, and enable the collection of release health metrics in the AndroidManifest.xml file.
+
+For more details, see the [full documentation on using Release Health with Android]({%- link _documentation/platforms/android/index.md -%}#release-health).
+
+### iOS
+To benefit from the health data provided in Releases v2, you must use at least version 5.0.0 Beta 2 of the Cocoa SDK and enable the collection of the release health metrics by adding `“enableAutoSessionTracking": true` into options during the initialization of the SDK.
```

Comment:
here we specify which field we should enable `enableAutoSessionTracking` but for Android we don't, as we already have a specific subpage for enabling it, I'd omit it and just link to the page, as we already do, so if by any chance we change that, we don't need to fix in several places.

Reply:
I have updated it.